docker-compose.yml

```diff
@@ -135,6 +133,19 @@ services:
     env_file:
         - .env
 
+  nginx:
+    image: bitnami/nginx
+    volumes:
+      - ./bilara_server_block.conf:/opt/bitnami/nginx/conf/server_blocks/bilara_server_block.conf:ro
+      - ./nginx_logs:/var/log/nginx
+      - ./frontend:/var/www/html:ro
+    ports:
+      - ${DOCKER_BACKEND_PORT}:${NGINX_PORT}
+    depends_on:
+      - bilara-backend
```

Comment:
During Bilara reboot, nothing is served from nginx until bilara-backed will start (which itself depend on elastic and data population)